bin/upload_sarif_to_github

```diff
@@ -64,4 +68,4 @@ curl -L \
   -H "Authorization: Bearer $GITHUB_TOKEN" \
   -H "X-GitHub-Api-Version: 2022-11-28" \
   --data-binary "@$sarif_json_temp_file" \
-  https://api.github.com/repos/"$owner"/"$repo"/code-scanning/sarifs
+  https://api.github.com/repos/"$slug"/code-scanning/sarifs
```

Comment:
I was going to suggest to [use our `github_api` helper from this plugin](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/blob/trunk/bin/github_api) while at it… but then looking at its code I just realized that has the similar issue of relying on `BUILDKITE_PULL_REQUEST_REPO` instead of `BUILDKITE_REPO`—and besides, the `awk` program it uses to attempt to extract the host if not provided is incorrect, as there's no field 4… so we'll need to fix that helper in a separate PR I guess, only at which point one could consider using it in your sarif helper implementation 🙃

Reply:
Thanks for checking this! I've added a task so we won't forget: https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/issues/125